draft-ietf-rats-corim.md

```diff
@@ -1103,11 +1103,10 @@ Environment.
 
 #### Endorsed Values Triple {#sec-comid-triple-endval}
 
-An Endorsed Values triple declares additional measurements that are valid when
-a Target Environment has been verified against reference measurements. For
-Endorsed Value Claims, the subject is either a Target or Attesting Environment,
-the object contains measurements, and the predicate defines semantics for how
-the object relates to the subject.
+An Endorsed Values triple declares additional claims that an Attester cannot declare about itself in the Evidence it produces.
+These claims are added to the ACS when its environment matches.
```

Comment:
```suggestion
These claims are added to the ACS.
```

Reply:
I think we concluded in the meeting that we ought to keep the matching requirement.